cpp/tests/io/parquet_reader_test.cpp

```diff
@@ -2556,6 +2566,20 @@ TEST_F(ParquetMetadataReaderTest, TestNested)
   auto meta = read_parquet_metadata(cudf::io::source_info{filepath});
   EXPECT_EQ(meta.num_rows(), num_rows);
 
+  auto const column_chunk_metadata = meta.columnchunk_metadata();
+  // Four leaf columns
+  EXPECT_EQ(column_chunk_metadata.size(), 4);
+  // Check if all leaf columns are present
+  EXPECT_TRUE(column_chunk_metadata.find("maps.key_value.key") != column_chunk_metadata.end());
```

Comment:
There's this option as well, but I'm not sure if it's even better here.
```suggestion
  EXPECT_NE(column_chunk_metadata.find("maps.key_value.key"), column_chunk_metadata.end());
```

Reply:
I think this one is ok as is since it's a pretty commonly occuring pattern with `find()`

Reply:
I think this one is ok as is since it's a pretty commonly occurring pattern with `find()`